notebook.ipynb

```diff
@@ -1,5 +1,12 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.            for k in range(72):
</pre><p>It would be beneficial for people who are unfimilar, if you included a comment explaining why 36 and 72 are used</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/CI2023-RC-team2/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
closing conversation, resolved